scss-c2s/c2s-results.scss

```diff
@@ -0,0 +1,379 @@
+@mixin media($size: false, $max-width: false, $min-width: false, $ignore-for-ie: false) {
+  @if $size == desktop {
+      @media (min-width: 925px) {
+        @content;
+      }
+    } @else if $size == tablet {
+      @media (min-width: 641px) {
+        @content;
+      }
+    } @else if $size == mobile {
+      @media (max-width: 640px) {
+        @content;
+      }
+    } @else if $max-width != false {
+      @media (max-width: $max-width) {
+        @content;
+      }
+    } @else if $min-width != false {
+      @media (min-width: $min-width) {
+        @content;
+      }
+    } @else {
+      @media (min-width: $size) {
+        @content;
+      }
+    }
+}
+
+$nhs-blue: #0D60B5;
+$blue: #669ed4;
+$grey: #E8EDEE;
+$grey-2: #bac4c8;
+$grey-3: #425563;
+$grey-4: #425563;
+$white: #fff;
```

Comment:
`$white` is available in `scss-live/environment/settings/_colours.scss`

Reply:
Lazy dev on my part. I've corrected.